### PR TITLE
docs(expressions): documenting stageByRef, currentStage, stageExists,…

### PR DESCRIPTION
**Pull request by marchello2000**
… pipelineId

### CODE REVIEW

File: reference/pipeline/expressions/index.md [hunk 1]
```diff
@@ -186,12 +186,36 @@ information from Jenkins properties files or other similar endpoints.
 
 ### #stage(String)
 
-A shortcut to get the stage by name. For example, `${#stage("Bake")}` allows
+A shortcut to get the stage by name. For example, `${#stage('Bake')}` allows
```
Comment (dorbin) [inline reference/pipeline/expressions/index.md]: @marchello2000 Any reason you changed `'` to `"` in this first occurrance, but not in subsequent occurrences in this paragraph?

File: reference/pipeline/expressions/index.md [hunk 2]
```diff
@@ -186,12 +186,36 @@ information from Jenkins properties files or other similar endpoints.
...
+Returns the current stage.
+
+### #stageExists(String)
+
+Checks if a given stage exists. You can search by `name` or `id`.
+Returns `true` if at least one stage is found with the `name` or `id` given.  
+Since the `id` is generated at runtime, most of the time it will make sense to search by `name` instead.
+Note that stage names are set by default so if you create a Webhook stage it will be called Webhook; 
+give the stage a unique name to be able to find it accurately.
+
```
Comment (dorbin) [inline reference/pipeline/expressions/index.md:211]: Suggest: "...giving the stage a unique name when you create it make it easier to find when using this helper function."

**Review by dorbin — CHANGES_REQUESTED** [review]
@marchello2000 
Just a few minor points/questions. 
And, are these helper functions new in Orca? Can you add a link here to the PR?

**Comment by marchello2000** [comment]
@dorbin Thanks!
Here are the PR links (most very recent, one quite old):
https://github.com/spinnaker/orca/pull/1884
https://github.com/spinnaker/orca/pull/2699
https://github.com/spinnaker/orca/pull/2718
https://github.com/spinnaker/orca/pull/2738

File: reference/pipeline/expressions/index.md (hunk 1 above)
Comment (marchello2000) [reply reference/pipeline/expressions/index.md]: my preference is `'` but I will just stick to convention and use `"` here

File: reference/pipeline/expressions/index.md (hunk 2 above)
Comment (marchello2000) [reply reference/pipeline/expressions/index.md:211]: nice, thanks!

Comment (dorbin) [reply reference/pipeline/expressions/index.md:211]: ignoring my typo, of course :wink: 
*"makes"

**Review by dorbin — APPROVED** [review]
Thanks @marchello2000

**Comment by marchello2000** [comment]
@dorbin since I am not an approver yet, do you mind merging the PR for me, please?

**Pull request merged**